Gulpfile.js

```diff
@@ -16,6 +16,15 @@ var config = {
       sprite: 'sprite.svg', //sprite name
       example: true // Build sample page
     }
+  },
+  svg: {
+    xmlDeclaration: false, // strip out the XML attribute
+    doctypeDeclaration: false, // don't include the !DOCTYPE declaration
```

Comment:
That's a great point about the attributes. I'm still pretty fresh in the SVG world, so I'm sure that there are plenty of ways to skin that SVG cat. :smile_cat: I wouldn't want to force developers into a certain corner with styling.

Want me to remove this object and re-commit?